CHANGELOG.md

```diff
@@ -1,6 +1,59 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.10.0 (2019-02-07)
+
+This release adds a `--registry-exclude-image` flag to the daemon to
```

Comment:
Reads a bit like the flag does all these things. I reckon:

This release adds the `--registry-exclude-image` flag for excluding images from scanning, allows for registries with self-signed certificates, and fixes several bugs.